examples/basics/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "__reframe_examples_basic",
   "private": true,
   "dependencies": {
-    "@reframe/react-kit": "0.0.1-rc.21"
+    "@reframe/react-kit": "0.0.1-rc.21",
+    "@brillout/fetch": "0.0.2"
```

Comment:
The example fails to find `brillout/fetch` otherwise. Still I didn't manage to run this example:

```
/usr/lib/node_modules/@reframe/cli/node_modules/webpack/lib/TemplatedPathPlugin.js:43
                                throw new Error(
          ^
Error: Path variable [contenthash] not implemented in this context: [name].style.hash_[contenthash].css
    at fn (/usr/lib/node_modules/@reframe/cli/node_modules/webpack/lib/TemplatedPathPlugin.js:43:11)
    at fn (/usr/lib/node_modules/@reframe/cli/node_modules/webpack/lib/TemplatedPathPlugin.js:31:16)
```

does it work for you @brillout ?

Reply:
If I remember correctly that's because webpack v4 doesn't have the `[contenthash]` variable anymore. I guess you are running webpack v3 and your cli is an older version.

On the beta versions almost whole reframe is contained into plugins and `@reframe/react-kit` is a "plugin-ensemble" that includes `@reframe/server` `@reframe/build` `@reframe/react` etc.

Running
```shell
$ npm remove -g @reframe/cli
$ npm install -g @reframe/cli@beta
```
should do the trick.

Reply:
It worked, thanks :) For the record I had to do

```
$ npm remove -g @reframe/cli
$ npm install -g @reframe/cli@beta
$ npm install
$ npm install @reframe/react-kit@beta
$ reframe start
```

Reply:
Ah yes and install the local deps